node/runtime/pangolin/src/bridges_message/pangolin_parachain.rs

```diff
@@ -147,23 +148,23 @@ impl ThisChainWithMessages for Pangolin {
 #[derive(Clone, Copy, RuntimeDebug)]
 pub struct PangolinParachain;
 impl ChainWithMessages for PangolinParachain {
-	type AccountId = bp_pangolin_parachain::AccountId;
```

Comment:
Why?

Reply:
Fixed. Please review it again. @AurevoirXavier @HackFisher